<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CertInfoDialog</class>
 <widget class="QDialog" name="CertInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>证书表查看</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>511</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>511</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>签名证书</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="horizontalSpacing">
       <number>30</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
         </font>
        </property>
        <property name="text">
         <string>颁发者</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="signerSerialLabel">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
         </font>
        </property>
        <property name="text">
         <string>0x 52048b9c8a67e28f0cc8cc75813ddc5a</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
         </font>
        </property>
        <property name="text">
         <string>序列号</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="signerIssuerLabel">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
         </font>
        </property>
        <property name="text">
         <string>VeriSign Class 3 Code Signing 2010 CA</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
         </font>
        </property>
        <property name="text">
         <string>颁发给</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="signerSubjectLabel">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
         </font>
        </property>
        <property name="text">
         <string>Tencent Technology(Shenzhen) Company Limited</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>时间戳证书</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="horizontalSpacing">
       <number>30</number>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="timeSerialLabel">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
         </font>
        </property>
        <property name="text">
         <string>0x 52048b9c8a67e28f0cc8cc75813ddc5a</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
         </font>
        </property>
        <property name="text">
         <string>颁发者</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
         </font>
        </property>
        <property name="text">
         <string>时间戳</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="timeSubjectLabel">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
         </font>
        </property>
        <property name="text">
         <string>Symantec Time Stamping Services Signer - G4</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
         </font>
        </property>
        <property name="text">
         <string>序列号</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="timeStampLabel">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>1234年12月3日 4时56分</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="timeIssuerLabel">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
         </font>
        </property>
        <property name="text">
         <string>Symantec Time Stamping Services CA - G2</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
         </font>
        </property>
        <property name="text">
         <string>颁发给</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CertInfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CertInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
